raaghu-elements/src/rds-progress-bar/rds-progress-bar.tsx

```
@@ -194,44 +195,43 @@
-                                                    name={"circle"} // Default icon if none provided
-                                                    fill={index%1===0}
-                                                    stroke={index%1===1}
-                                                    width={'20'}
-                                                    height={'20'}
-                                                />
-                                            )
+                                            <RdsIcon
+                                                name={"circle"} // Default icon if none provided
+                                                fill={index % 1 === 0}
+                                                stroke={index % 1 === 1}
+                                                width={'20'}
```

ESLint

[error] Strings must use doublequote.

(quotes)